Automattic/wp-calypso
# Launchpad: Add skip feature to the free flow

Related to:
* D120621-code
* https://github.com/Automattic/wp-calypso/pull/80661

## Proposed Changes

* Adds the Skip feature to the Free flow. This will allow users to skip the fullscreen launchpad and permit the user to access the Customer Home. Together with the D120621-code code, the users will also see the pre-launch launchpad on the Customer Home.

## Testing Instructions



* Apply D120621-code on your sandbox
* Navigate to `/setup/free``
* Go through the flow until you reach the launchpad screen
* Click on the `Skip for now` link on the top right corner
* You should be redirected to the Customer Home
* Click on the `Launch your site` task
* The Launchpad should disappear when you close the celebration modal

## Pre-merge Checklist
